app/src/main/java/com/github/libretube/ui/viewholders/InstancesViewHolder.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.libretube.ui.viewholders
```

Comment:
The class could be moved to the InstancesAdapter file.

Reply:
We currently have all the view holders for the recycler views in a separate file, so I don't really think we should do it different here? 🤔

Reply:
The Kotlin documentation [recommends](https://kotlinlang.org/docs/coding-conventions.html#class-layout) putting related declarations in the same file.

Reply:
You're right that, however I think that should be done in a separate PR for all RecyclerView adapters.
I guess it makes sense to do it for all view holders in a separate file until then to make it be the same everywhere.